draft-dekater-scion-dataplane.md

```diff
@@ -1913,10 +1923,10 @@ All fields within SIG Frame Header are in network byte order.
 - `Session ID` (8 bits) identifies a tunneling session between a pair of SIGs.
 - `Index` (16 bits) is the byte offset of the first beginning of an IP packet within the payload. If no IP packet starts in the payload, e.g. if the frame contains only the middle or trailing part of an IP packet, the field MUST be set to 0xFFFF.
 - `Reserved` (12 bits): it MUST be set to zero.
-- `Stream ID` (20 bits), along with the session, it identifies a unique sequence of SIG frames. Frames from the same stream are, on the remote SIG, put into the same reassembly queue. There may be multiple streams per session.
-- `Sequence Number` (64 bits) indicates the position of the frame within a stream. Consecutive frames are used to reassemble IP packets split among multiple frames.
+- `Stream ID` (20 bits), along with the session, it identifies a unique sequence of SIG frames. Frames from the same stream are, on the egress SIG, put into the same reassembly queue. There may be multiple streams per session.
+- `Sequence Number` (64 bits) indicates the position of the frame within a stream. Consecutive frames of a given stream have consecutive sequence numbers. IP packets split among multiple frames are re-assembled by concatenating the payloads of consecutive frames.
 
-SIGs MAY reorder received frames within a stream. Choice of frame reassembly window is left to implementations.
+A SIG MAY drop frames. Sorting frames received out-of-order by the egress SIG is optional. The egress SIG SHOULD drop frames from a stream if unable to perform the sequence re-assembly.
```

Comment:
My reason for this phrasing is to avoid ambiguity. Frames do have an order, defined by the sequence number. I want to be clear that the "re-ordering" we're talking about is to process them into *that* order and not changing it. Apparently that doesn't work. But I am not satisfied with just saying "reordering" I think it is not precise and relies on the audience guessing the meaning. May be the more specific "buffering frames received out-of-order is optional" would work?

Reply:
Sounds good, went for 
>  Buffering frames received out-of-order by the egress SIG is optional.